Documentation for notification service in Bring integration (#31522)
home-assistant/home-assistant.github.io: source/_integrations/bring.markdown

For authentication, the integration requires the `email` and `password` you used for your Bring! account. If you want to automatically receive notifications via the Bring! app when Home Assistant adds or removes an item from the list, you should use a dedicated account (such as `email: your.name+ha@gmail.com`) to connect Home Assistant with [Bring!](https://www.getbring.com/).

{% include integrations/config_flow.md %}

## Services

> You can use the services from the [to-do list](/integrations/todo/) to create, update or delete items on your Bring! shopping lists.

Reviewer: ```suggestion
You can use the services from the [to-do list](/integrations/todo/) to create, update, or delete items on your Bring! shopping lists.
```

### Notifications

The **Bring** integration offers a service to send push notifications to the Bring! mobile apps of other members of a shared shopping list. The Bring! mobile app has 4 predefined notification types. Note: If you want to receive these notifications yourself, you need to use a dedicated account as mentioned above.

| Service data attribute | Optional | Description                                                                                                                      |
| ---------------------- | -------- | -------------------------------------------------------------------------------------------------------------------------------- |
| `target`               |       no | Target Bring! list(s) whose members should be notified.                                                                          |

> | `message`              |       no | Type of push notification to send to list members. See [Notification types](#available-notification-types)                       |

Reviewer: ```suggestion
| `message`              |       no | Type of push notification to send to list members. See [Notification types](#available-notification-types).                      |
```

> | `item`                 |      yes | **Required for `urgent_message`.** Article name to include in in the message e.g.: *Urgent Message - Please buy cilantro urgently* |

Reviewer: ```suggestion
| `item`                 |      yes | **Required for `urgent_message`.** Article name to include in the message. For example: *Urgent Message - Please buy cilantro urgently*. |
```

> ### Available Notification types

Reviewer: ```suggestion
### Available notification types
```

> | Notification type | Text of Notification                                |

Reviewer: ```suggestion
| Notification type | Text of notification                                |
```
